recipes/clyngor/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "clyngor" %}
+{% set version = "0.3.30" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bca099abe4f97ad5f4b9270721b6047737414b14e5c50c1d4f97a2339e1b8cda
+
+build:
```

Comment:
could this be `noarch: python`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python